addons/danger/stringtable.xml

```diff
@@ -3,84 +3,111 @@
         </Key>
         <Key ID="STR_Lambs_Danger_Module_SetRadio_SettingToolTip">
             <English>Unit with radio toggled have boosted communications range when sharing information\nThis effect is also achieved by equipping the unit with a Vanilla Radio Backpack or TFAR-mod enabled radio.</English>
+            <German>Einheiten bei denen diese Option aktiviert ist haben eine erhöhte Kommunikationsreichweite.\n Dieser Effekt kann auch erzeugt werden durch das Ausrüsten der Einheit mit einem standard Radio-Rucksack oder einem TFAR-Mod kompatiblen Radio.</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Module_DisableGroupAI_DisplayName">
             <English>Configure Group AI</English>
+            <German>Gruppen-KI einstellen</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Module_DisableGroupAI_SettingName">
             <English>Disable LAMBS group AI</English>
+            <German>LAMBS KI für Gruppe ausschalten</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Module_DisableGroupAI_SettingToolTip">
             <English>Disables LAMBS group AI\nDisabling this feature prevents autonomous building assaults and clearing, as well as hiding from aircraft and tanks</English>
+            <German>Das Ausschalten verhindert das autonome Stürmen und Säubern von Gebäuden sowie das verstecken vor Panzern und Flugobjekten.</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_MainCat">
             <English>Settings</English>
+            <German>Einstellungen</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_GeneralCat">
             <English>General</English>
+            <German>Allgemein</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_ShareInformationCat">
             <English>Settings Share information</English>
+            <German>Informationen-Teilen Einstellungen</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_CQBFormationsCat">
             <English>Settings CQB Formations</English>
+            <German>CQB-Formationseinstellungen</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_Debug">
             <English>Debug</English>
+            <German>Debug</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_DisableDangerFSM">
             <English>Disable danger.fsm for player group</English>
+            <German>Danger.fsm für Spielergruppe deaktivieren</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_DisableDangerFSM_ToolTip">
             <English>Toggle advanced danger.fsm features on player group</English>
+            <German>Erweiterte danger.fsm Funktionen für Spielergruppe umschalten</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_DisableSuppressionPlayerGroup">
             <English>Disable suppression from player group</English>
+            <German>Niederhaltung für Spielergruppe deaktivieren</German>
```

Comment:
I did not serve in the military, so I am not sure which words are used in it. Basically I will just throw in what I would have used.

In this case instead of `Niederhaltung` what about something like `Unterdrückungsfeuer`?

Reply:
Niederhaltung is the effect of Unterdrückungsfeuer. So this is correct IMO.

Reply:
`Unterdrückungsfeuer für Spielergruppe deaktivieren` ist in diesem Fall jedoch richtig weil es hier nicht darum geht das die Einheiten unterdrückt werden, sondern das sie Unterdrücken.